storage.tf

```diff
@@ -0,0 +1,47 @@
+# Storage bucket for public files.
+
+resource "google_storage_bucket" "public" {
+  count = var.enable_storage ? 1 : 0
+
+  name                        = "${var.subdomain_prefix}storage.${var.top_domain}"
+  location                    = var.storage_bucket_location
+  storage_class               = var.storage_bucket_class
+  uniform_bucket_level_access = false
+
+  force_destroy = true
+
+  cors {
+    origin          = ["*"]
+    method          = ["*"]
+    response_header = ["*"]
+    max_age_seconds = 3600
+  }
+}
+
+resource "google_storage_bucket_access_control" "public" {
```

Comment:
If you navigate the domain of the storage bucket (e.g. `storage.examplepod.com`) it will list the files in the bucket. For many applications, this is not desirable. Is there a way to hide the file listings?